apps/docs/pages/guides/self-hosting/docker.mdx

```diff
@@ -220,3 +220,22 @@ Additional configuration is required for self-hosting the Analytics server. For
 export const Page = ({ children }) => <Layout meta={meta} children={children} />
 
 export default Page
+
```

Comment:
You are adding this below this, please reorder to have these exports at the bottom.

Reply:
🤦 will change thank you